src/content/en/2022/capabilities.md

```diff
@@ -16,7 +16,372 @@ featured_stat_2: TODO
+
+
+## Web Share API
+
+The [_Web Share API_](https://developer.mozilla.org/en-US/docs/Web/API/Web_Share_API) invokes the platform-specific sharing mechanism of the device, allowing data such as text, a URL, or files from a web application to be shared with any other application, such as mail clients, messaging applications, and more.
+
+The method called to share data is [`navigator.share()`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share). `navigator.share()` accepts an object containing the data to share and returns a Promise. Not every file type can be shared, though, and the [`navigator.canShare()`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/canShare) method can test a data object to see if the browser can share it.
+
+After calling `navigator.share()`, the browser will open a platform-specific sheet where users select which application to share the data with.
+
+Additionally, the Web Share API can only be triggered by a user's interaction with the page, such as a button click; the Web Share API cannot be called arbitrarily by executed code.
+
+```js
+const data = {
+  url: "https://almanac.httparchive.org/en/2022/capabilities",
+};
+
+if (navigator.canShare(data)) {
+  await navigator.share(data);
+}
+```
+
+The Web Share API is available on modern versions of Chrome, Edge, and Safari. For Chrome, though, it is only supported on Windows and Chrome OS.
+
+### Growth of Web Share API
+
+{{ figure_markup(
+  image="Web-Share-API-Usage.png",
+  caption="Usage of the Web Share API from 2021 to 2022 on desktop and mobile.",
+  description="The Web Share API shrunk in usage from 9.00% in 2021 to 8.84% in 2022 on desktop. On mobile, usage shrunk from 8.58% in 2021 to 8.36% in 2022.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTZp0g2lyspAvJUN-xV4TtqC_9wrRMqyg_bEzHCd1Be3p0Yhy3o2k-SH7DGX5a-LfaodNUTl4Ai-NXg/pubchart?oid=934956615&format=interactive",
+  sheets_gid="1887140434",
+  sql_file="fugu.sql"
+) }}
+
+The Web Share API shrunk in usage from 9.00% in 2021 to 8.84% in 2022 on desktop. On mobile, usage shrunk from 8.58% in 2021 to 8.36% in 2022. As a result, this year, the Web Share API was the second most used API on both desktop and mobile, falling behind the Async Clipboard API—last year's second most used API.
+
+On many sites, you can find the Web Share API in use. For example, social media platforms, documentation sites, and others use it as a great way to share content. Some examples where you can find the API in use include <a hreflang="en" href="https://web.dev/">web.dev</a> and [twitter.com](https://twitter.com/).
+
+{{ figure_markup(
+  gif="Web-Share-API.gif",
+  image="Web-Share-API.webp",
```

Comment:
Note: I added back the static webp version of all the gifs as well as the gifs. We show that the static version if someone has `prefers-reduced-motion` turned on, and for the ebook.